cip.go

```diff
@@ -193,7 +189,7 @@ func main() {
 			},
 		}
 	} else {
-		if *manifestPtr == "" && *manifestDirPtr == "" && *thinManifestDirPtr == "" {
+		if *manifestPtr == "" && *thinManifestDirPtr == "" {
 			klog.Fatal(fmt.Errorf("one of -manifest, -manifest-dir, or -thin-manifest-dir is required"))
```

Comment:
Nit: technically this is still true, but one of those is no longer an option :-)